### ISSUE TITLE
Scenario Result Chart is not rendering on Scenario Details tab

### ISSUE THREAD

**Issue by rcmedina** [trivago/cluecumber-report-plugin#44]
### Expected behavior
Scenario Result Chart graph should display on page

### Actual behavior
Scenario Result Chart graph is not displayed

### Reproduction steps
1. Open Cluecumber report, then click any test scenario link to open Scenario Details page

### Additional details (Maven version, Java version, etc.)
Cluecumber version 0.7.0

<img width="1430" alt="screen shot 2018-04-19 at 8 58 54 am" src="https://user-images.githubusercontent.com/37146992/38993086-a440f31c-43b0-11e8-8181-813824492bf8.png">

**Comment by laxersaz** [comment]
Hello! 

Which browser are you using to view it?
Do you see any js errors in the browser console?

Best, 
Benjamin

**Comment by rcmedina** [comment]
Hi @laxersaz 

Sorry for not replying back sooner.  Work got busy.

I investigated a bit more and the issue isn't browser-specific; it occurs in all browsers.

The issue occurs only when there is a Data Table at the Step-level...

**Below scenario will NOT render graph because there is a Data Table at Step-level:**

```
Scenario Outline: Live Chat - Required fields needed to initiate chat
    Given I login to external portal with <username> and <password>
    Then I am on Adjuster dashboard
    When I click the Live Chat button
    Then I am on Live Chat dialog
    When I select a chat reason
      | <reason> |
    And I type a chat question
      | <question> |
    Then I verify Start Chat button is enabled <assert>

    Examples:
      | username | password | reason          | question                  | assert |
      | medina   | test     | blank           | This is my Chat question? | false  |
      | medina   | test     | General Inquiry | blank                     | false  |
      | medina   | test     | General Inquiry | This is my Chat question? | true   |
```

**This scenario will render the graph because there is no Data Table at Step-level**

```
Scenario Outline: Search by First Name
    Given I login to external portal with <username> and <password>
    Then I am on Adjuster dashboard
    When I click the Patients menu
    Then I am on Search Patients List page
    And I enter First Name: <first_name>
    When I click Search button
    Then I verify the data returns the correct First Name: <first_name>

    Examples:
      | username | password | first_name |
      | medinar  | test     | John       |
```

**Comment by laxersaz** [comment]
Thanks a lot for the detailed description. I think I know why this happens and I will provide a fix soon.

**Comment by laxersaz** [comment]
On it now...

**Comment by laxersaz** [comment]
Fixed in 0.7.1
